tests/beta/test_default_tensor_tf.py

```diff
@@ -758,3 +758,39 @@ def circuit(x, w=None):
 
         with pytest.raises(ValueError, match="Device default.tensor.tf only supports diff_method='backprop' when using the tf interface"):
             res = cost_raising_error(params)
+
+
+class TestWiresIntegration:
+    """Test that the device integrates with PennyLane's wire management."""
+
+    def make_circuit_expval(self, wires, representation):
```

Comment:
Probs is not supported by this device, therefore only testing expval. (I am not sure if both is always necessary, but probs measurements involve a little bit of pipeline that expval does not... )

Reply:
iirc, the only extra pipeline for a device to support `return probs` is for the `Device.probability()` method be defined. This is defined and inherited automatically by devices inheriting `QubitDevice`, but not for `Device`.

Reply:
Do you think I should remove probability tests altogether and just have a single integration test using expval? In some sense, the rest should be tested in the ``device`` and ``qubit_device`` tests...